### DISCUSSION TITLE
Future of these web parts

### DISCUSSION THREAD

**Discussion by clayjam** [microsoft-search/pnp-modern-search#2978] [General]
Hi @wobba 
I use these web parts in Production and really like them as many others do. I am just a little concerned that we haven't seen a new release for 6 months when we previously having new releases every couple of months.
https://github.com/microsoft-search/pnp-modern-search/releases

Can you please let me know if the intention is to keep this project moving forward with regular releases and if Microsoft have an invested interest in this project or are the contributors all just part of the community and this project could cease at any stage? I'm am just getting a little concerned.

Thanks to all contributors for all your hard work - it is greatly appreciated.

**Comment by wobba** [comment]
As long as there is time to contribute, and we have people contributing, new versions will be made. No plans to shut it down. The current version is quite versatile and robust, but we aim to address bugs at the minimum. Also, Microsoft has an invested interest indeed as the parts are widely used by many organizations, and they are one of the top used 3rd party web parts in use.

My and @FranckyC's availability to contribute has been lower the past months, but it does not mean others can help out on issues they deem important.

**Reply by clayjam** [reply]
Thanks for the update Wobba - I really appreciate how active you are in this project. It is just concerning that if you or Franky are required elsewhere this may stop all together. Hopefully Microsoft would step in and allocate resources as the functionality these web parts provide are very important to businesses in the Modern search space.

**Reply by wobba** [reply]
If the community and customers want the parts to evolve, they should/could contribute development resources as well. That's how the project started and has evolved - adding features we needed ourselves for customers.

**Reply by clayjam** [reply]
I totally understand that this is a community project, however Microsoft left a large hole when it comes to search customisation from classic to modern. Many of us that use these web parts unfortunately don't have the skills to contribute from development perspective.

Custom search solutions that these web parts provide open up many benefits to find and consume content which only benefits the user experience and the platform, which is why Microsoft should have a vested interest - and because they haven't provided a Modern custom search solution.

Keep up the great work - it is very much appreciated.

**Reply by wobba** [reply]
I get your sentiment, and I wouldn't worry about the parts going away any time soon 🙂
